.github/workflows/update_debian_repo_script.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# to run this script as part of other update scripts, source the directory like below
+# source $GITHUB_WORKSPACE/.github/workflows/update_debian_repo_script.sh
+
+# you must have already set corresponding values for armhf_webPackages, arm64_webPackages, or all_webPackages
+# The corresponding appname in pi-apps will have its corresponding filepath= variable update
+# the filepath variable will contain the full filepath of the debian package with the version included
+
+# make sure required variable(s) is(are) supplied by the sourcing script
+if [ -n "$armhf_webPackages" ] || [ -n "$arm64_webPackages" ] || [ -n "$all_webPackages" ]; then
+#iterate through all variables supplied if an array
+# get longest arrary of passed variables
+num=$(echo -e "${#armhf_webPackages[@]}\n${#arm64_webPackages[@]}\n${#all_webPackages[@]}" | sort -n -r | head -1)
+
+for (( iter=0; iter<$num; iter++ )); do
+#make array appear as one variable inside for loop
+armhf_webPackages=${armhf_webPackages[$iter]}
+arm64_webPackages=${arm64_webPackages[$iter]}
+all_webPackages=${all_webPackages[$iter]}
+armhf_packagename=${armhf_packagename[$iter]}
+arm64_packagename=${arm64_packagename[$iter]}
+all_packagename=${all_packagename[$iter]}
+if [[ $iter == "0" ]]; then
+    version_number=""
+else
+    version_number=$((iter + 1))
+fi
+
+#set pi_apps_filepath variables
+if [ -n "$armhf_webPackages" ] && [ -f install-32 ]; then
```

Comment:
> `[ -f install-32 ]; then`

What would happen if the working directory was not set to the app folder?

Reply:
there would be issues if it wasn't setup as the app folder but now it is setup as the app folder and has been.

the `update_github_script.sh` script makes this same assumption